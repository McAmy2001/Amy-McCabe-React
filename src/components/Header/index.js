import React from 'react';
import Nav from '../Nav';

function Header(props) {
  return(
    <header className='header'>
      <h1 className='h1'><span>Amy </span><span>McCabe</span></h1>
      <Nav
        contentCategories={props.contentCategories}
        setCurrentContent={props.setCurrentContent}
        currentContent={props.currentContent}
      ></Nav>
    </header>
  )
}

export default Header;