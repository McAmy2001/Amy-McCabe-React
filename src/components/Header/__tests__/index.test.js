import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Header from '..';

afterEach(cleanup);

describe('Header component', () => {
  it('renders', () => {
    render(<Header></Header>)
  });
  it('matches snapshot', () => {
    const { asFragment } = render(<Header></Header>);
    expect(asFragment()).toMatchSnapshot();
  });
})