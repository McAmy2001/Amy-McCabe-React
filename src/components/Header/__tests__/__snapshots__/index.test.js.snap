// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component matches snapshot 1`] = `
<DocumentFragment>
  <header
    class="header"
  >
    <h1
      class="h1"
    >
      Amy McCabe
    </h1>
    <nav>
      <ul>
        <li>
          <a
            href="#about"
          >
            About Me
          </a>
        </li>
        <li>
          <a
            href="#portfolio"
          >
            Portfolio
          </a>
        </li>
        <li>
          Contact
        </li>
        <li>
          Resume
        </li>
      </ul>
    </nav>
  </header>
</DocumentFragment>
`;
